### PR TITLE
Feature: Video Page Completed

### PR DESCRIPTION
**Pull request by DejazmachMolla**
The completed/added requirements are listed below (to make your life easier :)).

Navigation bar
- It has an embedded image that represents a logo.
- The logo is aligned to the leftmost part of the navbar.
- It has an input tag that represents a search bar.
- The search bar is aligned to the middle of the navbar.
- It has a button tag that represents the search button.
- The search button is at the end (rightmost part) of the search bar.
- It has at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar.

Video
- It is displayed.
- It shows the video title.
- It shows the number of views.
- It has like and dislike icons.
- It has a share button.
- It shows the avatar of the user who uploaded the video.
- It has a subscribe button.

Sidebar Recommended video
- It shows the recommended video’s thumbnail preview (as image not as an actual video).
- It shows the title of the video.
- It shows the number of views.

### CODE REVIEW

**Review by davidmrtz-dev — CHANGES_REQUESTED** [review]
## Hi Team. 👨‍🚀  
### Congrats for your first project! You did an amazing job!
* Everything is positioned correctly 
* Your README file is clear and descriptive.
* Good Pull Request Message.
However, the following issues need to be fixed before moving on  👨‍💻  ⬇️ 

****

- [x] HTML Validator issues.

There are some issues displayed by the HTML Validator, kindly please fix this errors.

Screen Shot 2020-05-05 at 7 02 02 PM

****

- [x] Doesnt use id selector.

The overuse of the 'id' selector is not allowed, this is because this selector it's linked to use with Javascript, also it's considered a bad practice, here's an [article](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) where you can find more information about this topic.
It's preferred to use classes for styling, kindly please change the ids to classes.

****

- [x] Inline styling is not allowed.

The use of inline styling is not allowed because it's considered a bad practice, instead implement all the styling in its corresponding style sheet. Heres another [article](https://speckyboy.com/good-bad-css-practices) that points out why is this important.

****
### **Please kindly make the changes required and submit for re-review once you are ready!** 🙏 

****

### Status: Changes required. ♻️  👀 
****

Happy coding! 😄 💻  

**[TSE:  David Elí]** 👨‍💻 
Slack: @ David Eli Martinez Garcia

**Comment by alancphilpott** [comment]
@davitomix We have completed your suggested changes.

**Comment by santiago-dev96** [comment]
## Pull Request under review by @santiago-rodrig (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+                    </button>
+                    <img
+                        class="logo"
+                        src="https://archive.vn/Bss88/51a3832e6af6792add876216da444db23a4bea96.png"
+                        alt="Logo"
+                    />
+                </div>
+                <div class="search">
+                    <input type="text" placeholder="Search" />
+                    <button type="submit">
```
Comment (santiago-dev96) [inline index.html:36]: `<button></button>` has a `type="submit"` by default.
```suggestion
                    <button>
```

**Review by santiago-dev96 — APPROVED** [review]
# Project approved!

Congratulations, you've done an amazing job, the requirements are met, you can continue with the next projects. Have a nice day!

**Pull request merged**